base/cmd.jl

```diff
@@ -7,26 +7,30 @@ const UV_PROCESS_WINDOWS_VERBATIM_ARGUMENTS = UInt8(1 << 2)
 const UV_PROCESS_DETACHED = UInt8(1 << 3)
 const UV_PROCESS_WINDOWS_HIDE = UInt8(1 << 4)
 
+const RawCPUMask = Union{Nothing,Vector{Cchar}}
```

Comment:
declaring this to be signed or unsigned conditional on the OS seems odd.
```suggestion
const RawCPUMask = Union{Nothing,Vector{Bool}}
```

Reply:
Wouldn't it be nice to match this with the `jl_spawn` signature? (But maybe there should be a comment/docstring.)

Reply:
I actually removed this line (to make defining `show` easier) but I still mention `Cchar` elsewhere:

https://github.com/JuliaLang/julia/blob/ba5559f9d71a972ee15cbf29853651b97b887fa0/base/process.jl#L75-L77